lib/mix/lib/mix/tasks/new.ex

```diff
@@ -445,8 +445,8 @@ defmodule Mix.Tasks.New do
     use ExUnit.Case
     doctest <%= @mod %>
 
-    test "the truth" do
-      assert 1 + 1 == 2
+    test "salutates the world" do
```

Comment:
@binaryseed interesting! Actually google translator translates `salutate` properly to the spanish `saluda`, but wordreference.com doesn't seem to. It seems I've been using this word wrong for years without knowing 🌵 

I like `greets` tho, more informal.

Thanks!